packages/trpc/server/routers/viewer/slots/handleNotificationWhenNoSlots.ts

```diff
@@ -0,0 +1,123 @@
+import { Redis } from "@upstash/redis";
+
+import type { Dayjs } from "@calcom/dayjs";
+import { sendOrganizationAdminNoSlotsNotification } from "@calcom/emails";
+import { IS_PRODUCTION, WEBAPP_URL } from "@calcom/lib/constants";
+import { getTranslation } from "@calcom/lib/server";
+import { prisma } from "@calcom/prisma";
+
+type EventDetails = {
+  username: string;
+  eventSlug: string;
+  startTime: Dayjs;
+  visitorTimezone?: string;
+  visitorUid?: string;
+};
+
+// 7 days or 60s in dev
+const NO_SLOTS_NOTIFICATION_FREQUENCY = IS_PRODUCTION ? 604_800 : 60;
+
+const NO_SLOTS_COUNT_FOR_NOTIFICATION = 2;
```

Comment:
We will make these options configureable on a perOrg basis if required. 

Will allign with customer to find out what they want initally